template/src/pageComponents/mint/ContractDemo.tsx

```diff
@@ -68,7 +83,11 @@ export default function MintContractDemo() {
       <div className="flex-shrink-1 mt-10 w-full flex-grow-0 lg:mt-0">
         <h1 className="text-4xl font-bold">{collectionName}</h1>
 
-        <h2 className="my-5">{String(ethAmount)} ETH</h2>
+        {mintTxFeeEstimation && (
+          <h2 className="my-5">
+            Estimated tx fee: ~{mintTxFeeEstimation} {chain?.nativeCurrency.symbol ?? 'ETH'}
+          </h2>
+        )}
```

Comment:
Haven't played much with `wagmi` but we could be more descriptive here. Something like:

 ```suggestion
        <h2 className="my-5">
            {mintTxFeeEstimation 
              ?  `Estimated tx fee: ~${mintTxFeeEstimation} ${chain?.nativeCurrency.symbol ?? 'ETH'}`
              :  "Unable to estimate" }
        </h2>
```

Since `isLoadingFeeEstimate` is `false`, `mintTxFeeEstimation` should have a returned value.

Reply:
How about now?. I am not hiding the whole page now just because tx fee is loading, and added a simple loading spinner.